lua/efmls-configs/formatters/mix.lua

```diff
@@ -0,0 +1,18 @@
+-- Metadata
+-- languages: elixir
+-- url: https://github.com/astral-sh/ruff
```

Comment:
Thanks for the PR, can you change the URL to one linking to mix? It can be their github or a website that has the mix documentation.

Reply:
Done